apps/client/src/app/[user]/components/SearchStudyInput/index.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import {
+  inputStyle,
+  inputWrapper,
+} from "@/app/[user]/components/SearchStudyInput/index.css";
+import { IcnSearch } from "@/asset/svg";
+import { type KeyboardEvent, useRef } from "react";
+
+const SearchStudyInput = () => {
+  const inputRef = useRef<HTMLInputElement | null>(null);
+
+  const searchStudy = (event: KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === "Enter") {
+      //TODO: 검색 api 연결 (검색 페이지로 네비게이트)
+    }
+  };
```

Comment:
```suggestion
  const handleSearchStudy = (event: KeyboardEvent<HTMLInputElement>) => {
    if (event.key === "Enter") {
      //TODO: 검색 api 연결 (검색 페이지로 네비게이트)
    }
  };
```


이벤트 핸들러에 연결되는건 prefix로 `handle-`을 붙여주면 좋을 것 같아요!

Reply:
동의합니다!